pkg/cilium/types.go

```diff
@@ -26,11 +26,6 @@ const (
 	CiliumOperatorImageName = "cilium-operator"
 	// CiliumETCDOperatorImageName defines cilium etcd operator image name.
 	CiliumETCDOperatorImageName = "cilium-etcd-operator"
-	// CiliumNodeInitImageName defines cilium's node init image name.
-	CiliumNodeInitImageName = "cilium-nodeinit"
-	// CiliumPreflightImageName defines the cilium preflight image name.
-	CiliumPreflightImageName = "cilium-preflight"
-
```

Comment:
Can you please leave the empty line as delimiter between the cilium and hubble images?